<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="readButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>read</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setBackgroundButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>set background</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lockFocusButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>lock Focus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="autoFocusButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>autofocus</string>
   </property>
  </widget>
  <widget class="QListWidget" name="historyListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Double Click the correct name to add it to the list -&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="thresholdBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>21</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>E
X
P
O
R
T</string>
   </property>
  </widget>
  <widget class="QListWidget" name="collectedListWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>256</width>
     <height>201</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
